wordpress.org/public_html/wp-content/themes/pub/wporg-plugins/header.php

```diff
@@ -47,16 +61,50 @@
 						<button class="menu-toggle dashicons dashicons-arrow-down-alt2" aria-controls="primary-menu" aria-expanded="false" aria-label="<?php esc_attr_e( 'Primary Menu', 'wporg-plugins' ); ?>"></button>
 						<div id="primary-menu" class="menu">
 							<ul>
-								<?php
-								foreach ( $menu_items as $path => $text ) :
-									$class = false !== strpos( $_SERVER['REQUEST_URI'], $path ) ? 'active' : ''; // phpcs:ignore
+							<?php
+							foreach ( $menu_items as $path => $text ) : // phpcs:ignore
+								$class = false !== strpos( $_SERVER['REQUEST_URI'], $path ) ? 'active' : ''; // phpcs:ignore
 								?>
 								<li class="page_item"><a class="<?php echo esc_attr( $class ); ?>" href="<?php echo esc_url( home_url( $path ) ); ?>"><?php echo esc_html( $text ); ?></a></li>
-								<?php endforeach; ?>
-								<li><?php get_search_form(); ?></li>
+							<?php endforeach; ?>
+							<li><?php get_search_form(); ?></li>
 							</ul>
 						</div>
 					</nav><!-- #site-navigation -->
 				<?php endif; ?>
 			</div><!-- .site-branding -->
 		</header><!-- #masthead -->
+
+		<?php if ( $show_filter_bar ) : ?>
+		<div class="wporg-filter-bar">
+			<nav class="wporg-filter-bar__navigation" aria-label="<?php esc_html_e( 'Plugin filters', 'wporg-plugins' ); ?>">
+				<ul>
+				<?php
+				foreach ( $local_nav_items as $slug => $label ) {
+					$class = '';
+					if (
+						// URL contains this filter.
+						( $slug && str_contains( $_SERVER['REQUEST_URI'], "plugin_business_model=$slug" ) ) || // phpcs:ignore
+						// Set the All item active if no business model is selected.
+						( ! $slug && false === strpos( $_SERVER['REQUEST_URI'], 'plugin_business_model' ) ) // phpcs:ignore
+					) {
+						$class = 'is-active';
+					}
+					$url = home_url( '/' );
+					if ( $slug ) {
+						$url = add_query_arg( array( 'plugin_business_model' => $slug ) );
+					} else if ( ! is_home() ) {
+						$url = remove_query_arg( 'plugin_business_model' );
+					}
```

Comment:
This should be achievable just by using `get_term_link()` I'd have thought? We probably want to enable pretty-permalinks for these anyway, and/or use `/browse/community/`.

I'm happy to see the design/functionality launch as-is **before** that happens though, it can be resolved afterwards.

Reply:
I went with this so that we could easily build up filtered queries like `https://wordpress.org/plugins/browse/block/?plugin_business_model=commercial`, because I was under the impression that these are filters, not just static links (@jasmussen can confirm).